### PR TITLE
Updating Ratification Poll Templates

### PR DESCRIPTION
**Pull request by LongForWisdom**

### CODE REVIEW

**Review by prose11 — COMMENTED** [review]
LGTM!

Only comment is the sentence and paragraph summary were both in the template because MIP51 said they "must contain" both:
https://mips.makerdao.com/mips/details/MIP51#mip51c2-ratification-poll

**Comment by LongForWisdom** [comment]
> LGTM!
> 
> Only comment is the sentence and paragraph summary were both in the template because MIP51 said they "must contain" both:
> https://mips.makerdao.com/mips/details/MIP51#mip51c2-ratification-poll

Ugh, so it does, I missed that. That was probably silly of me to add.

File: governance/polls/templates/MIPs/Ratification Poll - $mip_code - $date_MONTH_DD,_YYYY.md [hunk 1]
```diff
@@ -21,29 +21,26 @@ The Governance Facilitators have placed a ratification poll into the [voting sys
...
 * The MIP Editors will mark the proposal **Accepted** and the Governance Facilitators will confirm its passage on the Governance and Risk call on $week_four_governance_call_date. 
-* The proposal will be added to the Maker Protocol, with implementation tasked to the relevant [Core Unit](https://mips.makerdao.com/mips/details/MIP38#mip38c2-core-unit-state) and [Domain Teams](https://mips.makerdao.com/mips/details/MIP7#mip7c2-the-current-domain-roles-list).
+* Any further work required to implement the proposal will be tasked to the relevant [Core Units](https://mips.makerdao.com/mips/details/MIP38#mip38c2-core-unit-state) and [Domain Teams](https://mips.makerdao.com/mips/details/MIP7#mip7c2-the-current-domain-roles-list).
 
-**If the votes for the Yes' option DO NOT exceed the votes for the 'No' option, OR less than 10,000 MKR vote in favor of this proposal, then this proposal will be marked as rejected per [MIP51](https://mips.makerdao.com/mips/details/MIP51#mip51c2-ratification-poll).**
+**Otherwise, this proposal will be marked as rejected per [MIP51](https://mips.makerdao.com/mips/details/MIP51#mip51c2-ratification-poll).**
```
Comment (blimpa) [inline governance/polls/templates/MIPs/Ratification Poll - $mip_code - $date_MONTH_DD,_YYYY.md:43]: The three "-" that come after turn this into a header, I think

See: https://github.com/makerdao/community/blob/a127f2ae83b4cea5365d9cf74ec98393b333adf9/governance/polls/templates/MIPs/Ratification%20Poll%20-%20%24mip_set%20-%20%24date_MONTH_DD%2C_YYYY.md

File: governance/polls/templates/MIPs/Ratification Poll - $mip_set - $date_MONTH_DD,_YYYY.md [hunk 2]
```diff
@@ -21,36 +21,50 @@ The Governance Facilitators have placed a ratification poll into the [voting sys
...
+Please review the links below to inform your position on this proposal before voting.
+* [Cannonical Proposal Version]($link_to_github_commit_version)
+* [Latest Proposal Version]($link_to_portal_version)
+* [Proposal Discussion Thread]($discussion_link)
+
+### $mip_code_3
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*($paragraph_summary)*
```
Comment (blimpa) [inline governance/polls/templates/MIPs/Ratification Poll - $mip_set - $date_MONTH_DD,_YYYY.md:52]: Are these parentheses necessary?

**Pull request merged**